background.js

```diff
@@ -21,12 +21,13 @@ class Background
      */
     draw(ctx)
     {
-        this.ctx.drawImage(AM.getAsset('./img/levels/backgroundTest.png'), this.x, this.y);
+        this.ctx.drawImage(AM.getAsset('./img/levels/extendedBackground.png'), this.x, this.y);
```

Comment:
We'll have to update background to take in a string path as a parameter instead of hardcoding, as we will need to call new on different levels. Nothing immediately though, this is ok for now.